drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/DroneWebDriverExtension.java

```diff
@@ -101,4 +92,20 @@ public void register(ExtensionBuilder builder) {
         builder.service(DroneInstanceEnhancer.class, AugmentingEnhancer.class);
         builder.observer(WindowResizer.class);
     }
+
+    private <T extends Configurator & Instantiator & Destructor> void registerFactoryService(
+        ExtensionBuilder builder, Class<? extends T> factory, String expectedDriver) {
```

Comment:
Is `? extends T` necessary when you use the factory parameter as the constraint itself so it can never be anything but `Class<T>`?